app/models/site_config.rb

```diff
@@ -81,6 +81,9 @@ class SiteConfig < RailsSettings::Base
   field :suggested_tags, type: :array, default: %w[beginners career computerscience javascript security ruby rails swift kotlin]
   field :sidebar_tags, type: :array, default: %w[help challenge discuss explainlikeimfive meta watercooler]
 
+  # Shop
+  field :shop_url, type: :string, default: "https://shop.dev.to"
```

Comment:
Correct me if i'm wrong, but I don't think it makes sense to have DEV specific defaults anymore. I believe we were planning on removing defaults in this file to avoid other communities having our defaults, and that way we can tell which fields are empty and show some sort of banner

Reply:
Note: If we remove this default, the DEV Team needs to make sure that we set SiteConfig. shop_url as https://shop.dev.to in the consols on production.

Reply:
I would keep defaults and then make a PR to remove defaults with a temporary script we can have the SRE team run in one go. What do you think?

Reply:
Sounds good to me 👍